rfc/rfc-0000-more-option-selectors.md

```diff
@@ -0,0 +1,104 @@
+# Meta
+[meta]: #meta
+- Name: More Option Selectors
+- Start Date: 2022-02-04
+- Author(s): Waciuma Wanjohi, Ciro Costa
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+RFC 09 introduces the use of the selector MatchFields in supply-chain resource options. These
+borrow their shape from
+[the selectors defined on Pods](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements).
+MatchFields can support matching on all (non-status) fields in the workload object. Carto
+options selector should also support the label specific `matchExpressions` and `matchLabels`.
+
+# Motivation
+[motivation]: #motivation
+
+Users may find the MatchLabels syntax simpler than the MatchFields syntax.
```

Comment:
```suggestion
Users may find the MatchLabels syntax simpler than the MatchFields syntax. MatchLabels is a standard across all k8s objects that have selectors, and MatchExpressions is implemented for Pods and Jobs, meaning these selectors might be more familiar to users.
```

Reply:
Both label and field selectors are widely used across k8s, but typically not combined into a single "selector". I broadly agree that because labels are fields, for this purpose a label selector may seem like syntactic sugar on top of a more generic field selector, but the historical uses are quite different.